src/test/unit/__snapshots__/profile-conversion.test.js.snap

```diff
@@ -489,12 +489,14 @@ Object {
         "time": Array [],
       },
       "name": "Chrome Thread",
+      "pausedRanges": Array [],
```

Comment:
This Google Chrome snapshot diff is concerning. Is this a bug?

Reply:
This comes from the change to `getEmptyThread`; the snapshot change is part of that commit too. It will look better once I extract this to its own PR.